dateutil/tz/tz.py

```diff
@@ -189,6 +208,22 @@ def __repr__(self):
 class tzlocal(_tzinfo):
     """
     A :class:`tzinfo` subclass built around the ``time`` timezone functions.
+
+    .. doctest:: tzlocal
+
+        >>> from datetime import *
+        >>> from dateutil.tz import *
+
+        >>> datetime.now(tzlocal())
+        datetime.datetime(2003, 9, 27, 10, 1, 43, 673605,
+                  tzinfo=tzlocal())
```

Comment:
flake8 would complain about indentation here and * imports

Reply:
True, though to be fair he's literally just copy-pasting from [here](https://dateutil.readthedocs.io/en/latest/examples.html#tzoffset-examples). Probably wouldn't be a terrible idea to go through the examples and fix them.

The problem now, though, is that we started literally copying stuff from the examples page into the individual module pages, because I didn't want to break old URLs. I realize now that the reason I didn't do that in the first place is that now we have to maintain the example documentation in *two places*.

I'm inclined to accept this PR since it's not breaking anything *new*, but to put a halt to these documentation moves until we work out a way to transclude examples.